src/app/student-division/committees/[slug]/_components/old-committee-card.tsx

```diff
@@ -46,13 +52,35 @@ export const OldCommitteeCard: FC<OldCommitteeCardProps> = ({
     >
       <AccordionItem className="border-none" value="image">
         <AccordionContent>
-          <CommitteeImage
-            alt={
-              image ? `Gruppbild på medlemmarna i ${name}` : `Logga på ${name}`
-            }
-            className="object-contain"
-            filename={image || logo}
-          />
+          <Dialog>
+            <DialogTrigger>
+              <CommitteeImage
+                alt={
+                  image
+                    ? `Gruppbild på medlemmarna i ${name}`
+                    : `Logga på ${name}`
+                }
+                className="object-contain"
+                filename={image || logo}
+              />
```

Comment:
Eftersom CommitteeImage använder Nexts Image "under the hood" så behöver vi skicka in en uppdaterad height och width för att bilden inte ska optimeras till något för litet och framstå som pixlig
```suggestion
              <CommitteeImage
                height={750}
                width={750}
                alt={
                  image
                    ? `Gruppbild på medlemmarna i ${name}`
                    : `Logga på ${name}`
                }
                className="object-contain"
                filename={image || logo}
              />
```